apps/portal-e2e/src/profile/hacker/index.spec.ts

```
@@ -0,0 +1,35 @@
+import { actorCalled, Task } from '@serenity-js/core'
+import { beforeEach, describe, it } from '@serenity-js/playwright-test'
+import { BrowseTheWeb, Navigate } from '@serenity-js/web'
+import { createProfile } from './abilities'
+
+export function startLoggedInNoProfile() {
+  return Task.where(
+    '#actor starts logged out',
+    Navigate.to('/profile'),
+  )
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)